test/unittest/security/authentication/CMakeLists.txt

```diff
@@ -61,7 +64,14 @@ target_include_directories(BuiltinPKIDH PRIVATE
     ${PROJECT_SOURCE_DIR}/include ${PROJECT_BINARY_DIR}/include
     ${PROJECT_SOURCE_DIR}/src/cpp
     )
-target_link_libraries(BuiltinPKIDH GTest::gtest ${OPENSSL_LIBRARIES} fastcdr foonathan_memory $<$<BOOL:${WIN32}>:ws2_32>)
+target_link_libraries(BuiltinPKIDH
+    GTest::gtest
+    ${OPENSSL_LIBRARIES}
+    fastcdr
+    foonathan_memory
+    $<$<BOOL:${WIN32}>:ws2_32>
+    $<$<BOOL:${LibP11_FOUND}>:eProsima_p11>  # $<TARGET_NAME_IF_EXISTS:eProsima_p11>
```

Comment:
```suggestion
    $<$<BOOL:${LibP11_FOUND}>:eProsima_p11>
```

Reply:
We are supporting a CMake version so old (3.11) that I like to hint how it should be done on a proper version.